blivet/devices/md.py

```diff
@@ -188,6 +193,35 @@ def level(self, value):
         self._level = level
 
     @property
+    def chunk_size(self):
+        if self.exists and self._chunk_size == Size(0):
+            self._chunk_size = self.read_chunk_size()
+        return self._chunk_size
```

Comment:
If someone accesses `chunk_size` while the array is inactive it will overwrite the potentially correct chunk size with `Size(0)`. The way we handle many things of this sort is to update the value whenever we know the array is active, eg: in `__init__` if the device is active and then again in `_post_setup`.